android/geowidget/src/main/java/org/smartregister/fhircore/geowidget/screens/GeoWidgetFragment.kt

```diff
@@ -285,12 +305,27 @@ class Builder {
         this.useGpsOnAddingLocation = value
     }
 
+    fun setMapLayers(list : List<MapLayer>) = apply {
+        this.mapLayers = list
+    }
+
+    fun setLocationButtonVisibility(show : Boolean) = apply {
+        this.shouldLocationButtonShow = show
+    }
+
+    fun setPlaneSwitcherButtonVisibility(show: Boolean) = apply {
+        this.shouldPlaneSwitcherButtonShow = show
+    }
```

Comment:
Can these be moved to some builder pattern or something? The settings can all be wrapped inside one data class and a single function is exposed to update the settings, instead of writing single-liner functions for each setting.

Reply:
Are the functions part of the `Builder` class?

Reply:
yes they all functions are part of Builder class